### PR TITLE
[ejector] chore: refactor exit order

### PR DESCRIPTION
**Pull request by vgorkavenko**

### CODE REVIEW

File: src/services/validator_state.py [hunk 1]
```diff
@@ -237,7 +242,16 @@ def get_recently_requests_to_exit_pubkeys(
 
         logger.info({'msg': f'Fetch exit events. Got {len(events)} events.'})
 
-        return set(bytes_to_hex_str(event['args']['validatorPubkey']) for event in events)
+        # Initialize dict with empty sets for operators which validators were not contained in any event
```
Comment (F4ever) [inline src/services/validator_state.py:245]: Why do we need this complication

File: src/utils/validator_state.py [hunk 2]
```diff
@@ -27,6 +28,19 @@ def is_on_exit(validator: Validator) -> bool:
     return int(validator.validator.exit_epoch) != FAR_FUTURE_EPOCH
 
 
+def is_exitable(validator: LidoValidator, last_requested_to_exit_index: int) -> bool:
```
Comment (F4ever) [inline src/utils/validator_state.py:31]: I'm not sure this belongs here.
last_requested_to_exit_index - it's our internal thing, this is business logic.
but here we store utils that can be used anywhere
Also we should mention one important thing that last_requested_to_exit_index works only if this validator is holding by node operator which last_requested_to_exit_index is.

Comment (vgorkavenko) [reply src/utils/validator_state.py:31]: That's why here is `LidoValidator` type for `validator` arg

File: src/services/validator_state.py (hunk 1 above)
Comment (vgorkavenko) [reply src/services/validator_state.py:245]: for the universality of uses

File: src/services/validator_state.py [hunk 3]
```diff
@@ -222,10 +225,12 @@ def non_exited_validators(validator: LidoValidator) -> bool:
 
         return validators_recently_requested_to_exit
 
-    def get_recently_requests_to_exit_pubkeys(
-            self, blockstamp: ReferenceBlockStamp,
-            chain_config: ChainConfig,
-    ) -> set[HexStr]:
+    def get_recently_requests_to_exit_indexes(
```
Comment (F4ever) [inline src/services/validator_state.py:228]: Rename

**Review by F4ever — APPROVED** [review]

**Pull request merged**